bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -366,8 +366,6 @@ def _validate_chunk_row_in_progress(self, chunk):
         """Helper for :meth:`_validate_chunk`"""
         assert self.state == self.ROW_IN_PROGRESS
         self._validate_chunk_status(chunk)
-        if not chunk.HasField('commit_row') and not chunk.reset_row:
-            _raise_if(not chunk.timestamp_micros or not chunk.value)
```

Comment:
The `protobuf` types don't use `None` as "emtpy", they just use the default values for their type, so setting `timestamp_micros = 0` is the same as not setting it at all